Test/ActivityPub.Types.Tests/Integration/Deserialization/MastodonInteropTests.cs

```diff
@@ -0,0 +1,269 @@
+        nestedActivity!.Actor.Should().NotBeEmpty();
+        nestedActivity!.Object.Should().NotBeEmpty();
+    }
+    
+    [Fact]
+    public void MastodonAcceptFollowActivityTest()
+    {
+        JsonUnderTest = """
+                        {
+                          "@context": "https://www.w3.org/ns/activitystreams",
+                          "type": "Accept",
+                          "actor": "https://host.example/actor/ahwPS_-DYE2h9RlTo0p4jg",
+                          "object": {
+                            "id": "http://mastodon.example/fbdfbd1c-489e-4edb-980f-0273a03c0992",
+                            "type": "Follow",
+                            "actor": "http://mastodon.example/users/user"
+                          }
+                        }
+                        """;
+        
+        ObjectUnderTest.Is<ASActivity>(out var activity).Should().BeTrue();
+        activity!.Actor.Should().NotBeEmpty();
+        activity!.Object.Should().NotBeEmpty();
+        
+        activity.Object!.First()!.Value!.Is<ASActivity>(out var nestedActivity).Should().BeTrue();
+        nestedActivity!.Actor.Should().NotBeEmpty();
+    }
+    
+    [Fact(Skip = "ContentMap not deserializing")]
+    public void MastodonUpateNoteActivityTest()
+    {
+        JsonUnderTest = """
+                        {
+                          "@context": [
+                            "https://www.w3.org/ns/activitystreams",
+                            {
+                              "atomUri": "ostatus:atomUri",
+                              "conversation": "ostatus:conversation",
+                              "inReplyToAtomUri": "ostatus:inReplyToAtomUri",
+                              "ostatus": "http://ostatus.org#",
+                              "sensitive": "as:sensitive",
+                              "toot": "http://joinmastodon.org/ns#",
+                              "votersCount": "toot:votersCount"
+                            }
+                          ],
+                          "actor": "https://mastodon.example/users/john",
+                          "cc": ["https://mastodon.example/users/john/followers"],
+                          "id": "https://mastodon.example/users/john/statuses/9876#updates/1675500855",
+                          "object": {
+                            "atomUri": "https://mastodon.example/users/john/statuses/9876",
+                            "attachment": [],
+                            "attributedTo": "https://mastodon.example/users/john",
+                            "cc": ["https://mastodon.example/users/john/followers"],
+                            "content": "<p>Creating test data 🆕</p>",
+                            "contentMap": {
+                              "en": "<p>Creating test data 🆕</p>",
+                              "es": "<p>Creando datos de prueba 🆕</p>"
+                            },
```

Comment:
I'm not sure if the `Content` property is supposed to be picking up `ContentMap` or not. But, it's not in this case

Reply:
This is an oversight on my part. I've documented it in #12 to be fixed ASAP.